helper/debug.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const _ = require('lodash');
+
+class Debug {
+  constructor(moduleName){
+    this.name = moduleName || 'unnamed module';
+  }
```

Comment:
I guess I'm feeling a little uneasy about this being a class, there's not really any reason for it to be so?

I think of classes as being a `'structure which contains state'` and that methods are used to `'operate on and modify internal state'`, in this case the state is being stored on the `req` object and so the functions are better off to be either:

1. next to the data (ie: as methods of `$req`)
2. as pure functions (ie. in a module somewhere, exposed as functions)